server/controllers/admin.controllers.js

```diff
@@ -0,0 +1,95 @@
+import model from '../models';
+import serverError from '../helpers/server-error';
+import validations from '../helpers/validations';
+
+const { User } = model;
+/**
+ *@class Admin
+ *@description Admin Controller
+ *@exports Admin
+ */
+/**
+ * @method activateReviewer
+ * @description Admin priviledges
+ * @param {Object} req
+ * @param {Object} res
+ * @returns {undefined}
+ */
+const activateReviewer = async (req, res) => {
+  // const adminId = req.user.userObj.id;
```

Comment:
If this code is commented out, we might as well remove it @amaechi-chuks

Reply:
Noted